modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/enricher/fixed/MetadataTypeEnricher.java

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.runtime.module.extension.internal.loader.enricher.fixed;
```

Comment:
fixed?

Reply:
better names are welcome ("static" cannot be the name of a package)